### DISCUSSION TITLE
Adding a KPS counter

### DISCUSSION THREAD

**Discussion by ItsShamed** [ppy/osu#19598] [Ideas] [answered]
It would be cool to have that in lazer as a skinnable component *like the pp counter* and also useful for some streamers so that they do not rely too much on third-party software. it should look like this:

https://user-images.githubusercontent.com/39100084/183229616-ecbb1c02-14f7-4961-98e6-3fe0f4df3277.mp4

So yeah, the implementation is already done, this is just to ask if it's okay for me to open a PR for it.

**Comment by peppy** [comment, marked as answer]
It's fine to open a full request for this. Note that it should handle all rulesets hopefully (with a single code path through depending on `UserInputManager` or the likes), but we can go through this in the pull request if it's not already the case.
